src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -371,6 +371,8 @@ pub enum Connector {
         broker: String,
         topic: String,
         key: Option<Vec<Ident>>,
+        consistency_topic: Option<String>,
+        consistency_topic_format: Option<Format<T>>,
```

Comment:
Tiny nit: it's probably worth having a `KafkaConsistency` struct that you can optionalize separately! That way you won't be able to represent a consistency topic format with no topic name:

```rust
enum Connector {
    Kafka {
        // ...
        consistency: Option<KafkaConsistency>
    }
}

struct KafkaConsistency {
    topic: String,
    topic_format: Option<Format<T>>
}
```

Reply:
(Er, that is, if it is true that you can't specify the format without specifying a topic.)

Reply:
That's true! Fixing